Xamarin.Forms.Core/Items/GridItemsLayout.cs

```diff
@@ -41,5 +41,9 @@ public double HorizontalItemSpacing
 			get => (double)GetValue(HorizontalItemSpacingProperty);
 			set => SetValue(HorizontalItemSpacingProperty, value);
 		}
+
+		internal class SpanSizeLookup
```

Comment:
say what ?

Reply:
I'm as confused as you are. I'll take out whatever that is.